subsys/zigbee/lib/zigbee_fota/src/dfu_multi_target.c

```diff
@@ -54,6 +57,15 @@ static struct dfu_image_writer dfu_multi_target_writer_1 = {
 };
 #endif
 
+#if CONFIG_UPDATEABLE_IMAGE_NUMBER > 2
+/** DFU target writer for image 1 (NET core). */
```

Comment:
```suggestion
/** DFU target writer for image 2 (external memory). */
```

Reply:
Small change in comments pushed, please verify :)

Reply:
Above seems unfixed